src/systems/userinput/bindings/windows-mixed-reality-user.js

```diff
@@ -0,0 +1,498 @@
+import { paths } from "../paths";
+import { sets } from "../sets";
+import { xforms } from "./xforms";
+import { addSetsToBindings } from "./utils";
+
+const v = name => {
+  return `/wmr-user/wmr-var/${name}`;
```

Comment:
We have typically been prefixing this with `/vars`. Fine for the paths to be whatever after that but would prefer not adding more roots. Doesn't really matter much now, but seems cleaner to keep the root namespace clean.